### ISSUE TITLE
Reification of :accompanier, :example in the Table -> switch domain and range?

### ISSUE THREAD

**Issue by flipz357** [amrisi/amr-guidelines#268]
The reification table listed in the guidelines maybe has two bugs (but I'm not sure). I think that

> x :accompanier y

should be translated as per my interpretation of [PropBank](https://verbs.colorado.edu/propbank/framesets-english-aliases/accompany.html) to 

> accompany a0 y a1 x

since a0 is the accompanier and a1 is the (ususally more passive) object.

Similarly, for

> x :example y

should be

> exemplify :a0 y a1 x

-----------------------------------------------

The other reifications that I checked seem okay, so far!

**Comment by nschneid** [comment]
https://www.isi.edu/~ulf/amr/lib/popup/accompanier.html gives some examples—I take it you're saying 

- "I ate pasta with tomato sauce" → tomato sauce/A0 accompanies pasta sauce/A1
- "The mayor arrived with a police escort" → police escort/A0 accompanies mayor/A1

That seems consistent with at least some examples in the data:

```
She arrived at the party without her husband.
  
(a / arrive-01
        :ARG1 (s / she)
        :ARG4 (p / party)
        :manner (a2 / accompany-01 :polarity -
              :ARG0 (p2 / person
                    :ARG0-of (h / have-rel-role-91
                          :ARG1 s
                          :ARG2 (h2 / husband)))
              :ARG1 s))
```

I.e. "her husband didn't accompany her". (BTW I'm not actually sure the `:manner` is actually needed there.)

Turning to predicative examples, the reification table gives "She's with him", which I'd say entails "She accompanies him" more than "He accompanies her".

A predicative example from the data:

```
Apparently, Franck has been with a prostitute who was under age.

(a3 / accompany-01
        :ARG0 (p / person :wiki "Franck_Ribéry" :name (n / name :op1 "Franck"))
        :ARG1 (p3 / person
              :ARG1-of (p2 / prostitute-01)
              :age (b / below
                    :op1 (t / temporal-quantity :quant 18
                          :unit (y / year))))
        :ARG1-of (a2 / appear-02))
```

Assuming the sentence is about accompaniment, the roles look correct to me.

I think that the issue is that for locations, the lower-numbered argument corresponds to x and would also be the subject in a predicative ("X is at Y."). Whereas with accompaniment, in attributive constructions the lower-numbered argument corresponds instead to y, but there is a sort of figure-ground reversal in predicative sentences ("X is with Y." means X accompanies Y).

**Comment by flipz357** [comment]
What I meant was that I think it's wrong only in the reification table in the guideline. I take it you understood me as saying that the annotation is kind of wrong? Sorry if my wording of this issue invited this interpretation. I will edit the title to include "table".

**Comment by nschneid** [comment]
I think it depends how you interpret the reification table. With respect to the example ("she's with him") it's correct: the subject is the lower-numbered argument and the predicative complement is the higher-numbered argument (i.e. 'she accompanies him'). With respect to the attributive use (`x :accompanier y`) it's flipped. Would it make sense to swap domain/range and change the example to "She is accompanied by him"?

**Comment by flipz357** [comment]
> Would it make sense to swap domain/range and change the example to "She is accompanied by him"?

Yes, I think so. Though I am not even sure if the example needs be changed, maybe? I took the example always as more of a hunch about what the relation in general is about.

My background was that Smatch++ uses the reification table to for de/reification of graphs for fairer comparison. For this, I don't use the examples, I just use the rules from the table. My problem can be seen here:

The guideline says:

> Read this chart as: x :Relation y = x :Domain-of (z / Reification :Range y)

So since for accompany we have:

> :accompanier       accompany-01       :ARG0       :ARG1        


we reify this graph:

```
(m / man
     :accompanier (c / cat))
```

The result per the rule is:

```
(m / man
     :arg0-of (a / accompany-01 
                   :arg1 (c / cat))
```

But I think it should be:

```
(m / man
     :arg1-of (a / accompany-01 
                   :arg0 (c / cat))
```

I see now that the similar issue has been fixed for the :poss relation recently. I think :example and :accompany can be handled similarly. Again, all other rules seem fine.
